src/javafundamentals/lambdas/Ex03Lambdas.java

```diff
@@ -10,6 +13,34 @@
 
 public class Ex03Lambdas {
     public static void main(String[] args) {
+        String name = "Arnold";
+
+        Tester tester = new Tester();
+
+        System.out.println(name);
+
+        tester.andyTest(inputName -> inputName.equals("Andy"),name);
+
+        name = tester.makeAndy(inputName -> {
```

Comment:
This is not the best usage for a function because you are ignoring the `input` is better for that purposes to use a `Supplier`.

Reply:
Yep, thanks